app/controllers/YourTaxCodeController.scala

```diff
@@ -87,7 +87,7 @@ class YourTaxCodeController @Inject()(
         val taxCodeViewModel =
           TaxCodeViewModelPreviousYears(taxCodeRecords, scottishTaxRateBands, year, applicationConfig)
         implicit val user: AuthedUser = request.taiUser
-        Ok(taxCodeDetailsPreviousYears(taxCodeViewModel))
+        Ok(taxCodeDetailsPreviousYears(taxCodeViewModel, request.fullName))
```

Comment:
Rather than adding another parameter you could also pass this information into the ViewModel

Reply:
ok have added here?: 
val taxCodeViewModel =
          TaxCodeViewModelPreviousYears(taxCodeRecords, scottishTaxRateBands, year, applicationConfig, request.fullName)